pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1472,7 +1470,9 @@ func (lb *LbaasV1) EnsureLoadBalancer(clusterName string, apiService *v1.Service
 			return nil, fmt.Errorf("Error creating floatingip for openstack load balancer %s: %v", name, err)
 		}
 
-		status.Ingress = append(status.Ingress, v1.LoadBalancerIngress{IP: floatIP.FloatingIP})
+		status.Ingress = []v1.LoadBalancerIngress{{IP: floatIP.FloatingIP}}
```

Comment:
Aside: I see this code is structured ever-so-slightly differently to the LbaasV2 version (with different corner cases around whether the floating IP already exists), which is unfortunate.  It would be nice if there was less duplication/variation here - otoh the LbaasV1 version could probably also be declared obsolete and just removed...

Reply:
Yeah, there are many different places Between LbaasV1 and LbaasV2, LbaasV1 has been  deprecated on OpenStack, so it is better to remove LbaasV1.